playwright-e2e/config/.env.staging

```diff
@@ -26,3 +26,13 @@ ANGIO_USER_PASSWORD=
 RGP_BASE_URL=https://rgp.staging.datadonationplatform.org
 RGP_USER_EMAIL=
 RGP_USER_PASSWORD=
+
```

Comment:
@aweng98 do we want that to point to dev or staging? currently pointing to dev

Reply:
`https://lms.dev.datadonationplatform.org` is wrong as this file is for Staging env. I'll fix this aone with others.